src/SwedbankPay.Sdk.Infrastructure/Consumers/ConsumersResponse.cs

```diff
@@ -0,0 +1,17 @@
+﻿using SwedbankPay.Sdk.Common;
+
+namespace SwedbankPay.Sdk.Consumers
+{
+    public class ConsumersResponse : IConsumersResponse
+    {
+        public ConsumersResponse(ConsumersResponseDto consumerResponse)
+        {
+            //Operations = consumerResponse.Operations.Map();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)